app/src/main/java/org/schabi/newpipe/fragments/list/kiosk/KioskFragment.java

```diff
@@ -161,4 +163,14 @@ public void handleResult(@NonNull final KioskInfo result) {
         name = kioskTranslatedName;
         setTitle(kioskTranslatedName);
     }
+
+    @Override
+    public void showEmptyState() {
+        // show "no live streams" for live stream kiosk
+        super.showEmptyState();
+        if (MediaCCCLiveStreamKiosk.KIOSK_ID.equals(currentInfo.getId())
+                && ServiceList.MediaCCC.getServiceId() == currentInfo.getServiceId()) {
+            setEmptyStateMessage(R.string.no_live_streams);
```

Comment:
This looks a bit difficult to maintain, but there is no better way atm I think. The optimal way would be for the extractor to announce what type of items a list will contain.